engines/cryomni3d/cryomni3d.cpp

```diff
@@ -130,7 +132,7 @@ Common::String CryOmni3DEngine::prepareFileName(const Common::String &baseName,
 
 void CryOmni3DEngine::playHNM(const Common::String &filename, Audio::Mixer::SoundType soundType,
 							  HNMCallback beforeDraw, HNMCallback afterDraw) {
-	const char *const extensions[] = { "hns", "hnm", nullptr };
+	const char *const extensions[] = { "hns", "hnm", "ubb", nullptr };
```

Comment:
As UBB is not supported at the moment I am not sure it's a good idea to add it.

Reply:
While it's still incomplete, I've implemented enough in the HNM decoder to allow the demo to run with audio only.

Reply:
Yes! I saw it and it's great. Thanks.